packaging/cloudify-logstash.spec

```diff
@@ -0,0 +1,99 @@
+
+Name:           cloudify-logstash
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+Summary:        Cloudify's Logstash
+Group:          Applications/Multimedia
+License:        Apache 2.0
+URL:            https://github.com/cloudify-cosmo/cloudify-amqp-influxdb
+Vendor:         Cloudify Platform Ltd.
+Packager:       Cloudify Platform Ltd.
+
+BuildRequires:  jre, rsync, logstash = 1:1.5.0
+Requires:       jre, postgresql94-jdbc
+Conflicts:      logstash
+
+Source0:        http://repository.cloudifysource.org/cloudify/components/logstash-output-jdbc-0.2.10.gem
+Source1:        http://repository.cloudifysource.org/cloudify/components/logstash-filter-json_encode-0.1.5.gem
+
+%define _user logstash
+
+# Work around lurking stale hashbangs from logstash's build process
+%define __requires_exclude ^/home/jenkins/.*
```

Comment:
shouldn't this be `__requires_exclude_from`? Just `__requires_exclude` makes it so that files in there won't be requirements; `_from` makes it so that files in there won't be scanned

Reply:
No. Some of the repackaged files from the original RPM have hashbangs which point to `/home/jenkins/...`